### PR TITLE
Add sds package

### PR DESCRIPTION
**Pull request by isaachier**

### CODE REVIEW

File: docs/packages/pkg/sds.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+.. index:: containers ; sds
+
+.. _pkg.sds:
+
+sds
+===
+
+-  `Official <https://github.com/antirez/sds>`__
+-  `Hunterized <https://github.com/hunter-packages/sds>`__
```
Comment (isaachier) [inline docs/packages/pkg/sds.rst]: I realize I should have done this upfront, but I still need a fork of this package.

Comment (ruslo) [reply docs/packages/pkg/sds.rst]: Added: https://github.com/hunter-packages/sds

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3006
* https://travis-ci.org/ingenue/hunter/builds/315843368?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3007
* https://travis-ci.org/ingenue/hunter/builds/315880387?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.191

**Pull request merged**